src/main/java/lotto/domain/WinningStatistics.java

```diff
@@ -38,7 +28,8 @@ private Prize matchTicketToPrize(LottoTicket lottoTicket, WinningLottoTicket win
 
 	public BigDecimal getWinningSummary() {
 		int ticketCount = prizeResult.size();
-		int revenue = prizeResult.stream().mapToInt(prize -> prize.getReward()).sum();
+		int revenue = prizeResult.stream()
+			.mapToInt(prize -> prize.getReward()).sum();
```

Comment:
stream을 사용한 함수형 코드를 작성할 때엔 위에서 아래로 읽을 수 있도록 한 줄에 하나의 함수만 사용해 직관적으로 코드를 작성하면 가독성을 향상시킬 수 있습니다.

```suggestion
		int revenue = prizeResult.stream()
			.mapToInt(Prize::getReward)
			.sum();
```